code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -449,7 +449,7 @@
 /datum/chemical_reaction/plastication
 	name = "Plastic"
 	result = null
-	required_reagents = list(/datum/reagent/acid/polyacid = 1, /datum/reagent/toxin/plasticide = 2)
+	required_reagents = list(/datum/reagent/acid/hydrochloric = 1, /datum/reagent/toxin/plasticide = 2)
```

Comment:
why are things requiring hydrochloric acid instead of triflic acid? the latter is what you've replaced polyacid with. This seems to make a lot of recipes wayyy easier to make

Reply:
Correct. Triflic Acid received major nerfs. Keep in mind that Triflic / Former Poly now requires a chemistry reagent you need Phoron for to make.

This meant I had to do a head scratch and consider in-depth whether existing stuff should use Triflic or just Hydrochloric. It became the latter, to make the Chemistry change lesser.